### PR TITLE
Create a general character ability system

### PR DESCRIPTION
**Pull request by CraZyB1336**
This pull request implements a general character movement as well as an ability system.

### CODE REVIEW

File: pexels-photo-413195.jpeg
Comment (ChrisKrane) [inline pexels-photo-413195.jpeg:1]: Nice grass 💯

File: pexels-photo-413195.jpeg.import [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+source_file="res://pexels-photo-413195.jpeg"
+dest_files=["res://.godot/imported/pexels-photo-413195.jpeg-63910991f4d174d89a72450ec495e892.s3tc.ctex"]
+
+[params]
+
+compress/mode=2
+compress/high_quality=false
+compress/lossy_quality=0.7
+compress/hdr_compression=1
+compress/normal_map=0
```
Comment (ChrisKrane) [inline pexels-photo-413195.jpeg.import:23]: Bad code example

Comment (CraZyB1336) [reply pexels-photo-413195.jpeg.import:23]: Sorry :3 uwu

File: pexels-photo-413195.jpeg
Comment (CraZyB1336) [reply pexels-photo-413195.jpeg:1]: thakns

**Review by Andrej-52 — APPROVED** [review]
Nice ψ(._. )>

**Pull request merged**